passes/equiv/equiv_simple.cc

```diff
@@ -374,7 +374,7 @@ struct EquivSimplePass : public Pass {
 				it.second.sort();
 
 				vector<Cell*> cells;
-				for (auto it2 : it.second)
+				for (const auto &it2 : it.second)
```

Comment:
This creates a reference to a pointer. How can that possibly be better than just copying the pointer?

Reply:
Same thing, it will collapse to just using the pointer directly.